### PR TITLE
Add Jenkinsfile

### PR DESCRIPTION
**Pull request by VinozzZ**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                                if(fileExists(".build/coverprofile")){
+                                    sh script: 'filter-cover-profile < .build/coverprofile > .build/clean.coverprofile', returnStatus: true
+                                    sh script: 'gocov convert .build/clean.coverprofile > .build/cover.json', returnStatus: true
+                                    sh script: 'gocov-xml  < .build/cover.json > .build/cobertura.xml', returnStatus: true
+                                    cobertura coberturaReportFile: '.build/cobertura.xml'
+                                }
+                            }
+                        }
+                    } */
+                // }
```
Comment (isaachess) [inline Jenkinsfile]: Why not just enable it now? And it'll run fast?

Comment (VinozzZ) [reply Jenkinsfile]: I thought it would complain about code coverage. I just uncommented it

Comment (isaachess) [reply Jenkinsfile]: Maybe try writing a single dummy test, so that it doesn't die on the test output?

Comment (isaachess) [reply Jenkinsfile]: I think it's better to just be running it, so that we don't have to turn it on in the future.

**Pull request merged**